TF_Service_dotNet/TouchFree_Service/Connection/ClientConnection.cs

```diff
@@ -269,38 +272,45 @@ protected void ProcessHandshake(ActionCode action, string requestContent)
             string clientApiVersion = (string)contentObj[VersionInfo.API_HEADER_NAME];
             Compatibility compatibility = GetVersionCompability(clientApiVersion, VersionInfo.ApiVersion);
 
+            string configurationWarning = string.Empty;
+
+            if (!configManager.AreConfigsInGoodState())
+            {
+                configurationWarning = " Configuration is in a bad state. Please update the settings via the Settings UI";
```

Comment:
This should probably be named 'TouchFree Settings' rather than 'Settings UI'

Reply:
I've updated this message